src/cases/containers/case-viewer-container/case-viewer-container.component.ts

```diff
@@ -1,20 +1,63 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import { CaseView } from '@hmcts/ccd-case-ui-toolkit';
+import { CaseTab, CaseView } from '@hmcts/ccd-case-ui-toolkit';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { select, Store } from '@ngrx/store';
+import { combineLatest } from 'rxjs';
+import { Observable } from 'rxjs/Observable';
+import { map } from 'rxjs/operators';
+import { AppConstants } from '../../../app/app.constants';
+import { UserDetails } from '../../../app/models/user-details.model';
+import * as fromRoot from '../../../app/store';
 
 @Component({
   selector: 'exui-case-viewer-container',
   templateUrl: './case-viewer-container.component.html',
   styleUrls: ['./case-viewer-container.component.scss']
 })
 export class CaseViewerContainerComponent implements OnInit {
+  private static WORKALLOCATIONRELEASE1 = 'WorkAllocationRelease1';
+  private static WORKALLOCATIONRELEASE2 = 'WorkAllocationRelease2';
```

Comment:
_Nitpick:_
Subject for a different conversation and I know it's simply been moved, but... ugh! 🤢

Reply:
What do you suggest here @paul-graham ?

Reply:
Well, it's kind of awful that they're there in the first place - they should be declared somewhere that _anywhere_ that needs to toggle on the version of Work Allocation being used can get at them. In addition, there's no casing and they're also mutable so not really constants. I'd go with something more like:
```
public static readonly FEATURE_WORK_ALLOCATION_RELEASE_1 = 'WorkAllocationRelease1';
public static readonly FEATURE_WORK_ALLOCATION_RELEASE_2 = 'WorkAllocationRelease2';
```

And I've put them as `public` because I think they should be elsewhere. I'm assuming `'WorkAllocationRelease1` and `WorkAllocationRelease2` are LaunchDarkly values or something.

Reply:
I am going to leave them as private because there aren't being used anywhere else. However they're there because we are checking which feature is activated in launch darkly. If its WorkAllocationRelease2 the pretended tabs should be shown.

Reply:
Ah, you've got to love a **pretend** tag. 😂 

No problem, with respect to them remaining `private` if this is really the only place those constants are needed. Thanks for at least giving them better names. 🙂